examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -65,26 +84,29 @@ NativeStackTrace::NativeStackTrace(uint32_t pid, const unsigned char *raw_stack,
     goto out;
   }
 
+  struct bcc_symbol resolved_symbol;
```

Comment:
Declared here but not used here - declare it inside the loop, where it's actually used?

Reply:
Agreed.